embc-app/ClientApp/src/app/home/home.component.scss

```diff
@@ -1,80 +1,90 @@
-@import "~src/styles/_defaults.scss";
+// @import "~src/styles/_defaults.scss";
 
-$container-width: auto; // 775px
+// $container-width: auto; // 775px
 
-// ==> cards
-.picture-card {
-  max-width: $card-width;
-  background-color: $card-body-bg;
-  padding-bottom: 48px;
-}
-
-// ==> home
+.card-img-top {
+  width: 100%;
+  height: 15vw;
+  object-fit:scale-down;
 
-.home-margins {
-  max-width: 1200px;
-  // padding-left: 30px;
-  // padding-right: 30px;
 }
+.card {
+  background-color: rgba(241, 244, 250, 1);
 
-.btn-primary {
-  font-family: "myriad-pro", "sans serif";
-  font-weight: 700;
-  font-style: normal;
-  font-size: 14px;
 }
+// // ==> cards
```

Comment:
Please delete obsolete code. If we want to find it again in the future then we'll look at commits in the branch history (which are hopefully titled well enough to know what they changed).